pkg/apis/build/v1alpha2/cluster_lifecycle_validation.go

```diff
@@ -0,0 +1,30 @@
+package v1alpha2
+
+import (
+	"context"
+
+	"knative.dev/pkg/apis"
+
+	"github.com/pivotal/kpack/pkg/apis/validate"
+)
+
+func (cl *ClusterLifecycle) SetDefaults(context.Context) {
+	// TODO: confirm, should we add something here?
```

Comment:
Thoughts?

Reply:
Is this where we specify the image to use if the lifecycle CRD is not configured?